src/log.js

```diff
@@ -150,7 +151,7 @@ class Log extends GSet {
    * @returns {Entry|undefined}
    */
   get (hash) {
-    return this._entryIndex[hash]
+    return this._entryIndex[hash] || null
```

Comment:
In fact, I've changed this because of `strictEqual`. One of the tests was checking against null instead of undefined so I've changed it to pass the test and forgot it there. Should I change the test to check for undefined instead? (It kinda makes sense to me since in this case we're indicating the absence of a value for that hash in the index instead of it being undefined. The interpretation here might vary though ¯\_(ツ)_/¯ )

Reply:
Got it! 👍 Let's remove the null and check against undefined in the test.